cap_anndata/cap_anndata.py

```diff
@@ -290,3 +290,20 @@ def obs_keys(self) -> List[str]:
 
     def var_keys(self) -> List[str]:
         return self.var.column_order.tolist()
+
+    def create_repr(self) -> str:
+        indent = " " * 4
+        s = f"CapAnnData object"
```

Comment:
Please add one more line. 

```suggestion
        s = f"CapAnnData object"
        s += f"\n{indent}File: {self._file}"
```